spec/support/media_spec_helpers.rb

```diff
@@ -0,0 +1,65 @@
+module MediaSpecHelpers
+  module FileHandling
+    def media_fixture(name)
+      path = File.expand_path("../../fixtures/media/#{name}", __FILE__)
+      File.open(path)
+    end
+  end
+
+  shared_context 'media helpers' do
+    def factory_name(described_class)
+      described_class.name.underscore.gsub('/', '_').to_sym
+    end
+
+    def file_for_type(file_type)
```

Comment:
If this method is only used to get fixtures for the reject case, perhaps you could remove it and just pass `mp3`, `jpg`, etc. directly to the shared example.